domino_environments/_environments.py

```diff
@@ -197,9 +200,14 @@ def deployment_version(self):
         return self.request_manager.get(url).json().get("version")
 
     def refresh_defaults(self):
+        self._environments_list = self.get_environments_list()
         self._default_environment = self.get_default_environment()
         self._default_details = self.get_revision_details(self._default_environment)
 
+    def get_environments_list(self):
+        url = self._routes.get_environments_list()
```

Comment:
Updating the usage from prior change.
```suggestion
        url = self._routes.environments_list()
```